### DISCUSSION TITLE
Question on XPath (Web Scraping) Issue in my feeds...

### DISCUSSION THREAD

**Discussion by quella** [FreshRSS/FreshRSS#6001] [Q&A] [answered]
I'm going to start by saying that I have been recently playing more with Web Scraping due to a few sites I visit do not provide any RSS feeds, and I'm glad the tool supports other options for sites like these.  Currently, I think I have a good process to get the item details extracted when I open an item that has been scraped.  However, I seem to be hitting one issue and I'm not sure which element it relates to in the XPath configuration.  So, any direction or help explaining (point me to a resource), I would be thankful.

In article reader view, the titles, content, topics all show the same item, but inside the item, it is correct?  Is this because of the "Content" field.  I assumed it was "Article Title" but this does not seem to be the case when I validate it via Google Chrome inspect.

![Screenshot 2023-12-29 at 12 01 24 PM](https://github.com/FreshRSS/FreshRSS/assets/823352/5c067c01-552d-4f24-ac8f-deacc9d03291)

When I open an article, I see the correct information as shown below.

![Screenshot 2023-12-29 at 12 01 49 PM](https://github.com/FreshRSS/FreshRSS/assets/823352/1110f2ca-ea1c-4bb0-afb2-7c7aa14b203a)

For the Makerworld feed shown in the above, I use the following XPath setup.  Again, everything seems to work and for the three configured XPath sites, I get the same behavior.  That is why I think I'm missing something.  Thoughts?

![Screenshot 2023-12-29 at 12 48 14 PM](https://github.com/FreshRSS/FreshRSS/assets/823352/e096a12a-7f78-494b-8fb6-b0e2c134baff)

Thank you all for all you do.  Quella.

**Comment by quella** [comment, marked as answer]
Did some more playing and found my issue.  It was that I had the "//" at the start of the relative path objects.  I removed them, did more validation and all is good.  What a great tool!

**Reply by Alkarex** [reply]
Nice to hear :-)

**Reply by quella** [reply]
Thank you all for the tool I use all the time an now opening up my world to Web Scraping where I can.  FreshRSS just keeps on getting better and better.
